source/intel.js

```diff
@@ -594,7 +594,7 @@ function NeptunesPrideAgent() {
     let arrival = new Date(now.getTime() + msplus);
     let p = prefix !== undefined ? prefix : "ETA ";
     let ttt = p + ampm(arrival.getHours(), arrival.getMinutes());
-    if (arrival.getDay() !== now.getDay())
+    if (arrival.getDay() != now.getDay())
```

Comment:
Why Remove it here but add it above.

Reply:
This case could have been left as !== as it's correct, but I must have missed that fact as I was trying to apply as few of these changes as possible given that this lint error totally borked the code.